edx_exams/apps/core/models.py

```diff
@@ -18,9 +21,10 @@ class User(AbstractUser):
 
     """
     full_name = models.CharField(_('Full Name'), max_length=255, blank=True, null=True)
+
     lms_user_id = models.IntegerField(null=True, db_index=True)
 
-    anonymous_user_id = models.IntegerField(null=True, db_index=True)
+    anonymous_user_id = models.UUIDField(default=uuid.uuid4, db_index=True, unique=True)
```

Comment:
Should I add any tests?

Reply:
the only thing to test really is that django's field definition works which isn't worth testing